doc/demos/NODDI_lowres.md

```diff
@@ -0,0 +1,95 @@
+# Decreasing Resolution
+
+This tutorial illustrates how to **drecrease the resolution of the look-up table (a.k.a LUT)** in AMICO.
+
+## Starting Script
+
+Check the **NODDI_01** demo from the following [link](https://github.com/daducci/AMICO/blob/master/doc/demos/NODDI_01.md), which contains an example dataset and the following code:
+
+```python
+import amico
+
+amico.core.setup()
+
+ae = amico.Evaluation("Study01", "Subject01")
+
+amico.util.fsl2scheme("Study01/Subject01/NODDI_protocol.bval", "Study01/Subject01/NODDI_protocol.bvec")
+
+ae.load_data(dwi_filename = "NODDI_DWI.img", scheme_filename = "NODDI_protocol.scheme", mask_filename = "brain_mask.img", b0_thr = 0)
+
+ae.set_model("NODDI")
+ae.generate_kernels()
+
+ae.load_kernels()
+
+ae.fit()
+
+ae.save_results()
+```
+
+## Change the resolution
+
+In order to change the resolution of the LUT in AMICO, we have to specify the desired number of directions by changing the value of the parameter `ndirs` in the functions:
+
+- amico.core.setup()
+- ae.generate_kernels()
+
+In this example, we modify the above code to use 500 directions. The modified code should be like this:
+
+```python
+import amico
+
+amico.core.setup( ndirs=500 )
+
+ae = amico.Evaluation("Study01", "Subject01")
+
+amico.util.fsl2scheme("Study01/Subject01/NODDI_protocol.bval", "Study01/Subject01/NODDI_protocol.bvec")
+
+ae.load_data(dwi_filename = "NODDI_DWI.img", scheme_filename = "NODDI_protocol.scheme", mask_filename = "brain_mask.img", b0_thr = 0)
+
+ae.set_model("NODDI")
+ae.generate_kernels( ndirs=500 )

```

Comment:
If some kernels were created before, maybe for a different number of directions, the program continues and uses those kernels.
It crashes if more directions are used that in the previous experiment